src/Compilers/CSharp/Test/Symbol/Symbols/UnsignedRightShiftTests.cs

```diff
@@ -1869,5 +1937,1583 @@ static void Main()
+        }
+
+        [Fact]
+        public void UserDefined_02()
+        {
+            // The IL is equivalent to: 
+            // public class C1
+            // {
+            //     public static C1 operator >>>(C1 x, int y)
+            //     {
+            //         System.Console.WriteLine("">>>"");
+            //         return x;
+            //     }
+            // }
+
+            var ilSource = @"
+.class public auto ansi beforefieldinit C1
+    extends [mscorlib]System.Object
+{
+    .method public hidebysig specialname static 
+        class C1 op_UnsignedRightShift (
+            class C1 x,
+            int32 y
+        ) cil managed 
+    {
+        .maxstack 8
+
+        IL_0000: ldstr "">>>""
+        IL_0005: call void [mscorlib]System.Console::WriteLine(string)
+        IL_000a: ldarg.0
+        IL_000b: ret
+    }
+
+    .method public hidebysig specialname rtspecialname 
+        instance void .ctor () cil managed 
+    {
+        .maxstack 8
+
+        IL_0000: ldarg.0
+        IL_0001: call instance void [mscorlib]System.Object::.ctor()
+        IL_0006: ret
+    }
+}
+";
+
+            var source1 =
+@"
+class C
+{
+    static void Main()
+    {
+        Test1(new C1(), 1);
+    }
+
+    static C1 Test1(C1 x, int y) => C1.op_UnsignedRightShift(x, y); 
+}
+";
+
+            var compilation1 = CreateCompilationWithIL(source1, ilSource, options: TestOptions.DebugExe,
+                                                       parseOptions: TestOptions.RegularPreview);
+            // PROTOTYPE(UnsignedRightShift): This code was previously allowed. Confirm that we are Ok with this 
+            //                                breaking change and document it.  
+            compilation1.VerifyDiagnostics(
+                // (9,40): error CS0571: 'C1.operator >>>(C1, int)': cannot explicitly call operator or accessor
+                //     static C1 Test1(C1 x, int y) => C1.op_UnsignedRightShift(x, y); 
+                Diagnostic(ErrorCode.ERR_CantCallSpecialMethod, "op_UnsignedRightShift").WithArguments("C1.operator >>>(C1, int)").WithLocation(9, 40)
+                );
+        }
+
+        [Fact]
+        public void UserDefined_03()
+        {
+            var source1 = @"
+public class C1
+{
+    public static void operator >>>(C1 x, int y)
+    {
+        throw null;
+    }
+
+    public static void operator >>(C1 x, int y)
+    {
+        throw null;
+    }
+}
+
+public class C2
+{
+    public static C2 operator >>>(C1 x, int y)
+    {
+        throw null;
+    }
+
+    public static C2 operator >>(C1 x, int y)
+    {
+        throw null;
+    }
+}
+
+public class C3
+{
+    public static C3 operator >>>(C3 x, C2 y)
+    {
+        throw null;
+    }
+
+    public static C3 operator >>(C3 x, C2 y)
+    {
+        throw null;
+    }
+}
+
+public class C4
+{
+    public static int operator >>>(C4 x, int y)
+    {
+        throw null;
+    }
+
+    public static int operator >>(C4 x, int y)
```

Comment:
Consider adding a positive case for parameters being nullable: `public static int operator >>>(S1? x, int? y)`